### PR TITLE
Miscellaneous fixes extracted from the work on SSF

### PR DESCRIPTION
**Pull request by elia**
## Summary

Just a few housekeeping changes, probably the only worth mentioning is the `--migrate` option for the installer.

## Checklist

Check out our [PR guidelines](https://github.com/solidusio/.github/blob/master/CONTRIBUTING.md#pull-request-guidelines) for more details.

The following are mandatory for all PRs:

- [x] I have written a thorough PR description.
- [x] I have kept my commits small and atomic.
- [x] I have used clear, explanatory commit messages.

The following are not always needed (~cross them out~ if they are not):

- [x] I have added automated tests to cover my changes.
- [ ] I have attached screenshots to demo visual changes.
- [ ] I have opened a PR to update the [guides](https://github.com/solidusio/edgeguides).
- [ ] I have updated the README to account for my changes.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,11 +3,8 @@ version: 2.1
 orbs:
   browser-tools: circleci/browser-tools@1.1
 
-  # Always take the latest version of the orb, this allows us to
-  # run specs against Solidus supported versions only without the need
-  # to change this configuration every time a Solidus version is released
-  # or goes EOL.
-  solidusio_extensions: solidusio/extensions@volatile
+  # Stick to the current major, so we won't get disrupting updates
+  solidusio_extensions: solidusio/extensions@0
```
Comment (kennyadsl) [inline .circleci/config.yml:7]: Do we still need this? Is the orb broken?

**Review by kennyadsl — APPROVED** [review]
Left one question, but not blocking 👍

File: .circleci/config.yml (hunk 1 above)
Comment (elia) [reply .circleci/config.yml]: Well, I have mixed feelings, "volatile" is really discouraged, but we're also in pre-1.0 mode… I'll revert for now, and we'll think of releasing 1.0 on the ORB and handle changes more "semantically" after that.

Comment (kennyadsl) [reply .circleci/config.yml]: Agreed

**Pull request merged**